src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Machine {
+    public void doWork() {
+        System.out.println("Machine started it is work");
+    }
+    public void stopWork() {
+        System.out.println("Machine stopped it is work");

```

Comment:
Incorrect grammar in the message. Change 'it is' to 'its'. Example: 'Machine stopped its work'